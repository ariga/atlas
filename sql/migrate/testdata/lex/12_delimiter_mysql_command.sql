delimiter //
create table t2 (a int) //

delimiter ;
delimiter //

create table t3 (a int) //
delimiter ;

show tables;
drop table t2, t3;
