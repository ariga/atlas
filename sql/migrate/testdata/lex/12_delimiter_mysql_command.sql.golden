create table t2 (a int)
-- end --
create table t3 (a int)
-- end --
show tables;
-- end --
drop table t2, t3;